docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,100 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` procedure prepares an Iceberg table for moving or copying to another location.
+
+### `rewrite-table-path`
+
+Stages a copy of the Iceberg table's metadata files where every absolute path source prefix is replaced by the specified target prefix.  
+This can be the starting point to fully or incrementally copy an Iceberg table located under an absolute path under a 
+source prefix to another under the target prefix.
+
+!!! info
+    This procedure only prepares metadata or/and data files for an existing Iceberg table for copying or moving to a new location.
```

Comment:
Only delete files are copied, right?

Reply:
Not exactly, to fully replicate the table from one existing location to another, all files referenced by given iceberg table need to move, which include all of metadata and data files. The rewrite table path only rewrite the necessary files which have the path reference

Reply:
I'm confused. Does it copy data files to staging folder?

Reply:
No it does not, this is just first step of three: rewrite, copy and re-register. Rewrite focus on iceberg metadata which have path reference, think about metadata.json, manfiest-list, manifest and position delete files, those all contain a file pointer in its content which require rewrite path for. Copy is the second step, where we can copy the files needed to the target. The copy target is superset of rewritten files, also include the data files.

Reply:
How about:

`` This procedure only stages rewritten metadata files and prepares a list of files to copy.  The actual file copy is not part of this procedure.```

Reply:
Adding copy + move everywhere in the doc is a bit awkward, I think it is not a common case to 'mv' the file as it destroys the original table.  Even if so, I think the user will get it and we dont have to make the doc awkward for that?